File: hotel-server/src/main/java/eu/getsoftware/hotelico/service/booking/application/customer/common/dto/UserDTO.java
// path: hotelico/service/booking/application/customer/common/dto/UserDTO.java

package eu.getsoftware.hotelico.service.booking.application.customer.common.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * For serialization to JSON!
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserDTO {

    private long entityId;

    private String title;

    private String firstName;

    private String lastName;

    private String email;

    private String company;

    private String jobTitle;

    // private List<String> languages;

    // private boolean authLinkedIn;

    private String hotelDomainId;
}

File: hotel-app/src/main/kotlin/eu/getsoftware/hotelico/customer/common/dto/UserDTO.kt
package eu.getsoftware.hotelico.customer.common.dto;

/**
 * for serialisation to JSON!
 */
class UserDTO
{
    private var id: Int = 0

    private var title: String? = null

    private var firstName: String? = null

    private var lastName: String? = null

    private var email: String? = null

    private var company: String? = null

    private var jobTitle: String? = null

//    private List<String> languages;

//    private boolean authLinkedIn;

    private var hotelId: Int? = null
}
